app/workers/asset_manager_create_attachment_asset_worker.rb

```diff
@@ -0,0 +1,51 @@
+class AssetManagerCreateAttachmentAssetWorker < WorkerBase
+  include AssetManager::ServiceHelper
+
+  # Carrierwave runs on an after_save hook and the transaction that inserts Assetable into the database
+  # might not be committed yet. This can cause a race condition where the worker runs before the assetable is readable.
+  # Use TransactionAwareClient for this worker to ensure that the commit is finished before the worker is executed.
+  sidekiq_options queue: "asset_manager", client_class: Sidekiq::TransactionAwareClient
+
+  def perform(temporary_location, asset_params, attachable_model_class, attachable_model_id, draft = false, auth_bypass_ids = [])
```

Comment:
I'm thinking the correct default for draft should be true for `AttachmentWorker`. Though hard to tell since we're catering for both the `PolicyGroup` and `Edition` cases. I believe you have plans to break this further.